lib/split/experiment.rb

```diff
@@ -17,6 +18,7 @@ class Experiment
     DEFAULT_OPTIONS = {
       :resettable => true,
       :retain_user_alternatives_after_reset => false,
+      :cohorting_block => ["control", "alternative"]
```

Comment:
I put a default value of ["control", "alternative"] here, we can set this to whatever we want. The problem currently is that if someone is using alternatives other than 'control' and 'alternative' the default won't work and they will have to override it